config/routes.rb

```diff
@@ -17,6 +17,11 @@
     resources :questions
     post 'vote', to: 'votings#vote'
   end
+
+  resources :groups do
```

Comment:
There is no need to replicate the `groups` namespace, you can just use the existing one.